cloud_AWS/terraform/module/cloudexport.tf

```diff
@@ -7,22 +7,22 @@ terraform {
 }
 
 resource "kentik-cloudexport_item" "aws_export" {
-  # Create only when plan_id is set
-  count = var.plan_id == "" ? 0 : 1
+  # Create when bucket_name_list or plan_id is set
+  count = var.bucket_name_list != null ? length(var.bucket_name_list) : var.plan_id == "" ? 0 : 1
 
-  name           = "${var.name}-${terraform.workspace}" # cloudexport name must be unique
+  name           = var.bucket_name_list != null ? "${var.name}-${terraform.workspace}${count.index}" : "${var.name}-${terraform.workspace}" # cloudexport name must be unique
   type           = "CLOUD_EXPORT_TYPE_KENTIK_MANAGED"
   enabled        = var.enabled
   description    = var.description
   plan_id        = var.plan_id
   cloud_provider = "aws"
   aws {
-    bucket = join(", ", [
+    bucket = var.bucket_name_list != null ? "${var.bucket_name_list[count.index]}/" : join(", ",[
       for bucketobject in aws_s3_bucket.vpc_logs :
       (var.s3_flowlogs_path == "" ? bucketobject.bucket : "${bucketobject.bucket}/${var.s3_flowlogs_path}")
     ])
     iam_role_arn      = var.create_role ? aws_iam_role.kentik_role[0].arn : ""
-    region            = var.region
+    region            = var.regions != null ? var.regions[count.index] : var.region
```

Comment:
Is it possible to have just one input variable "regions" instead of "regions" and "region"?

Reply:
Still to do